### PR TITLE
Books service permission

### PR DESCRIPTION
**Pull request by KirillDrago**

### CODE REVIEW

**Review by LiudmylaKulinchenko — APPROVED** [review]

File: library/views.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from rest_framework import viewsets
+from rest_framework.permissions import IsAdminUser, AllowAny
```
Comment (shurushku) [inline library/views.py]: Do you use AllowAny?

File: user/views.py [hunk 2]
```diff
@@ -4,21 +4,15 @@
 from rest_framework.authtoken.views import ObtainAuthToken
 from rest_framework.settings import api_settings
```
Comment (shurushku) [inline user/views.py]: You don't use TokenAuthentication, ObtainAuthToken and api_settings as i see

**Review by shurushku — CHANGES_REQUESTED** [review]

File: library/views.py (hunk 1 above)
Comment (KirillDrago) [reply library/views.py]: Nope...

**Review by shurushku — APPROVED** [review]
🔥🔥🔥

**Pull request merged**